### DISCUSSION TITLE
how to know which client came in server channel

### DISCUSSION THREAD

**Discussion by deahhh** [GenieFramework/Genie.jl#652] [Q&A]
```julia
using Genie.Router
Genie.config.websockets_server = true

channel("/hello/hey";) do 
      clientId = ??
      @show clientId
end
```

how to code "??" ?

THANKS!

**Comment by essenciary** [comment]
`params()` from `Genie.Router` (automatically exported) provides access to the request context. You should have all the relevant info there, check it out.

**Reply by deahhh** [reply]
Thanks, I'll try it now!
